src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -437,12 +437,14 @@ robocopy /S /NP %EXECUTION_DIR%int\%(Identity)\ %EXECUTION_DIR%native\
       <!-- On Windows, call prevents the test command from making execution end prematurely -->
       <TestCommandLines  Condition="'$(TargetOS)'=='Windows_NT' and '$(TestGCStressLevel)' != ''" Include="set COMPlus_GCStress=$(TestGCStressLevel)"/>
       <TestCommandLines  Condition="'$(TargetOS)'=='Windows_NT' and '$(OuterLoop)' == 'true'" Include="set XUNIT_PERFORMANCE_MIN_ITERATION=1"/>
-      <TestCommandLines  Condition="'$(TargetOS)'=='Windows_NT' and '$(OuterLoop)' == 'true'" Include="set XUNIT_PERFORMANCE_MAX_ITERATION=1"/>
+      <TestCommandLines  Condition="'$(TargetOS)'=='Windows_NT' and '$(OuterLoop)' == 'true'" Include="set XUNIT_PERFORMANCE_MAX_ITERATION=2"/>
```

Comment:
Can we instead have this to be set to a property that is defaulted to 1 but can be overridden? That way, we can only have it be two on the tests where we need it, but keep it 1 for the rest of the cases.

Reply:
> That way, we can only have it be two on the tests where we need it, but keep it 1 for the rest of the cases.

In what case would we want to keep it at 1?

Reply:
Well, all cases only required 1 iteration before, so I believe we shouldn't change all of them to be 2 now only because of some tests that need to run twice. That is why I would set the default as 1 so most tests will continue to only execute once, and special case the ones we know that need to run 2 times.

Reply:
The change to 2 is not test specific. None of the test require 2 iterations. Changing to 2 allows us to verify the correctness of the test run (i.e. data is being produced), which I believe is part of the intention of running the performance tests briefly as part of outerloop. If that is not important, I could revert all back to 1 (since there isn't any requirement, from the tests perspective, to run more than that).

Reply:
To unblock CI for now, I could keep the change minimal (and revert back to 1 iteration). Would that be preferable?

Reply:
I mean the other reason why I wanted to have it parameterized is that that way we don't require a change in buildtools if we want to change it in the future. I know that it is not test-specific but instead this is a setting that gets set per test project.